recipes/server_check.rb

```diff
@@ -4,6 +4,10 @@
 
 consul_service_folder = '/etc/systemd/system/consul.service.d'
 drop_in_file = "#{consul_service_folder}/server_start_prevention.conf" # prevent systemd from starting
+grafana_agent_folder = '/etc/grafana-agent/textfiles' # text based metric for grafana agent to push
+metric_file = "#{grafana_agent_folder}/consul_server.prom"
+metric_name = 'consul_server_allowed_to_start' # grafana agent will add the labels such as datacenteripaddress
+metric_help = 'A boolean variable indicating whether this consul server should be allowed to start.'
```

Comment:
Super nit - prometheus doesn't have booleans. This is a gauge type.

https://prometheus.io/docs/concepts/metric_types/#gauge